metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -243,15 +403,16 @@ def get_platform_from_database_id(self, database_id):
             return "athena"
         if platform_name == "clickhousedb":
             return "clickhouse"
-        if platform_name == "postgresql":
-            return "postgres"
```

Comment:
Why remove this?

Reply:
Hey Semion!
Thanks for catching this, I think we were rebasing and somehow this change didn't make it through. Will look to add this back in